src/panels/logbook/ha-logbook.ts

```diff
@@ -58,6 +62,18 @@ class HaLogbook extends LitElement {
     `;
   }
 
+  private async _fetchUsers() {
+    const users = await fetchUsers(this.hass);
+    users.forEach((user) => {
+      this._userid_to_name[user.id] = user.name;
```

Comment:
It doesn't update now after the users are loaded

Reply:
You could do this is a temporary object and after the forEach set it to `this._userid_to_name` that will make sure we do a re-render.

Reply:
I'm not following.  Is the concern that we remove users and then the user ids are still in `this._userid_to_name` ?

Reply:
No, LitElement only re-render if a property is changed, it does this with a strict check, if we add things to `this._userid_to_name` it will not change `this._userid_to_name`. 

So the LitElement will not re-render, thus not showing names until something else changed that will make it rerender.

Reply:
This btw would only be a problem is the logbook data is loaded before the user data, that is probably never going to happen but you are well on your way...